src/frontend/src/lib/components/hero/Actions.svelte

```diff
@@ -41,6 +42,9 @@
 
 	let isTransactionsPage = false;
 	$: isTransactionsPage = isRouteTransactions($page);
+
+	let sendAction = false;
+	$: sendAction = $anyBalanceNonZero || isTransactionsPage;
```

Comment:
Is this correct, your PR description does not mention an exception or am I missing something?

Reply:
the requirement should be to be hidden ONLY in the main page (not the transaction page), but ONLY when there is at least a token with non-zero balance

you are right, it was not clear in the description: I updated it